exercises/wordy/src/lib.rs

```diff
@@ -1 +1,16 @@
+pub struct WordProblem;
 
+impl WordProblem {
+    pub fn new(command: &str) -> Self {
+        unimplemented!(
+            "From the '{}' command construct a new WordProblem struct.",
+            command
+        );
+    }
+
+    pub fn answer(&self) -> Option<i32> {
+        unimplemented!(
+            "Return the result of the transmitted command, or an error, if the command is invalid."
```

Comment:
I think this should talk about `None` instead of `error`, now, right?

I'll prefer that change be amended into the Result -> Option commit so all the necessary changes for Result->Option are in one commit. It's small enough that amending won't cause a review burden.

Reply:
Agreed.